deliver/lib/deliver/upload_metadata.rb

```diff
@@ -451,10 +459,43 @@ def retry_if_nil(message, tries: 5, wait_time: 10)
       end
     end
 
-    # Finding languages to enable
-    def verify_available_info_languages!(options, app, languages)
-      app_info = fetch_edit_app_info(app)
+    # Checking if the metadata to update includes localised App Info
+    def updating_localised_app_info?(options, app, app_info)
+      app_info ||= fetch_live_app_info(app)
+      unless app_info
+        UI.important("Can't find edit or live App info. Skipping upload.")
+        return false
+      end
+      localizations = app_info.get_app_info_localizations
```

Comment:
[Unrelated to this PR]

@rogerluan we definitively need to improve our YARD doc on many of our model methods. This isn't clear at all from the definition of the `AppInfo#get_app_info_localizations` what that method is supposed to return, which makes it harder to read code that uses it like here and why Lore needed to use `localizations.find` below.

_(After some digging, the answer is that it returns `Array<Spaceship::ConnectAPI::AppInfo>`; which makes sense in retrospect I guess, but was't obvious to me at first sight, and RubyMine isn't able to deduce it automatically)_